runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterPlugin.java

```diff
@@ -377,6 +391,79 @@ else if (containsBoulder) // For traps like deadfalls. This is different because
 		lastTickLocalPlayerLocation = client.getLocalPlayer().getWorldLocation();
 	}
 
+	@Subscribe
+	public void onInteractingChanged(InteractingChanged interactingChanged)
+	{
+		Actor target = interactingChanged.getTarget();
+		Actor source = interactingChanged.getSource();
+
+		mapActorToChinTarget(target, source);
```

Comment:
Since there's no other references to the source and target, there's no need to save them to variables.
```suggestion
		mapActorToChinTarget(interactingChanged.getTarget(), interactingChanged.getSource());
```